src/v3/src/components/Widget/style.scss

```diff
@@ -0,0 +1,294 @@
+  position: relative;
+  min-inline-size: 300px;
+  border-bottom: map.get($ods-tokens, 'border', 'width', 'main') solid map.get($ods-tokens, 'border', 'color', 'display');
+}
+.okta-container .applogin-banner .applogin-background {
+  background-color: #fff;
+  opacity: 0.9;
+  position: absolute;
+  inset: 0;
+  box-shadow: 0 0 2px 1px hsla(0, 0%, 68.6%, 0.3);
+}
+.okta-container .applogin-banner .applogin-container {
+  position: relative;
+  inline-size: 400px;
+  min-inline-size: 300px;
+  margin-block: 0;
+  margin-inline: auto;
+  padding-block: map.get($ods-tokens, spacing, '3') map.get($ods-tokens, spacing, '4');
+  padding-inline: 0;
+  box-sizing: border-box;
+  text-align: center;
+}
+.okta-container .applogin-banner .applogin-container p {
+  /* TODO: OKTA-586564 replace hardcoded value */
+  color: #6e6e78;
+  margin-block: 0;
+}
+
+@media only screen and (max-width: 400px) {
+  .okta-container .applogin-banner .applogin-container {
+    inline-size: 100%;
+  }
+}
+@media only screen and (max-height: 750px) {
+  .okta-container .applogin-banner .applogin-container {
+    padding-block: 10px;
+    padding-inline: 0;
+  }
+}
+@media only screen and (max-height: 660px) {
+  .okta-container .applogin-banner .applogin-container {
+    padding-block: 5px;
+    padding-inline: 0;
+  }
+  .okta-container .applogin-banner .applogin-container p {
+    display: none;
+  }
+}
+.okta-container .applogin-banner .applogin-container h1 {
+  margin-block: 0;
+}
+.okta-container .applogin-banner .applogin-app-title {
+  display: none;
+}
+.okta-container .applogin-banner .applogin-app-logo {
+  display: inline-block;
+  vertical-align: middle;
+  margin-block-end: map.get($ods-tokens, 'spacing', '2');
+}
+.okta-container .applogin-banner .applogin-app-logo img {
+  width: map.get($ods-tokens, 'spacing', '8');
+  height: map.get($ods-tokens, 'spacing', '8');
+}
+
+/* loginpage container styles */
+.auth .content {
+  min-block-size: 100%;
+  min-inline-size: 300px;
+  display: inline-block;
+  inline-size: 100%;
+}
+.auth .content:after {
+  content: "";
+  display: block;
+  block-size: 30px;
+}
+.login-bg-image {
+  background-repeat: no-repeat;
+  background-position: 50%;
+  background-size: cover;
+  position: fixed;
+  inset: 0;
+  z-index: -5;
+}
+@media only screen and (max-width: 600px) {
+  .login-bg-image {
+    background-image: none !important;
+    background-color: #fff !important;
+    -webkit-filter: unset !important;
+    filter: unset !important;
+  }
+}
+
+/* signin container styles */
+#signin-container {
+  margin-block: map.get($ods-tokens, 'spacing', '8');
+}
+@media only screen and (max-width: 600px) {
+  #signin-container {
+    margin-block: map.get($ods-tokens, 'spacing', '2');
+  }
+}
+
+/* back to settings button style */
+.back-to-settings {
+  margin: unset !important;
+  min-height: unset !important;
+  width: unset !important;
+  min-width: unset !important;
```

Comment:
Update it in `loginpage.theme.css` can affect styles for loginpage that still stay with gen2, as the theme styles comes after widget styles, right now I didn't find a good approach to prevent gen2 related theme styles.